general/g.findfile/main.c

```diff
@@ -104,8 +104,10 @@ int main(int argc, char *argv[])
             strcpy(name, file_opt->answer);
         G_free_tokens(map_mapset);
     }
-    else
-        strcpy(name, file_opt->answer);
+    else {
+        strncpy(name, file_opt->answer, sizeof(name) - 1);
```

Comment:
Better use the new `G_strlcpy()` function.

Reply:
Done changes. Could you please review it once